aws/resource_aws_api_gateway_documentation_part_test.go

```diff
@@ -0,0 +1,243 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/apigateway"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSAPIGatewayDocumentationPart_basic(t *testing.T) {
+	var conf apigateway.DocumentationPart
+
+	rString := acctest.RandString(8)
+	apiName := fmt.Sprintf("tf_acc_api_%s", rString)
+	properties := `{\"description\":\"Terraform Acceptance Test\"}`
+	uProperties := `{\"description\":\"Terraform Acceptance Test Updated\"}`
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSAPIGatewayDocumentationPartDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccAWSAPIGatewayDocumentationPartConfig(apiName, properties),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSAPIGatewayDocumentationPartExists("aws_api_gateway_documentation_part.test", &conf),
+					resource.TestCheckResourceAttr("aws_api_gateway_documentation_part.test", "location.#", "1"),
+					resource.TestCheckResourceAttr("aws_api_gateway_documentation_part.test", "location.0.type", "API"),
+					resource.TestCheckResourceAttr("aws_api_gateway_documentation_part.test", "properties", `{"description":"Terraform Acceptance Test"}`),
```

Comment:
Can this use `strconv.Unquote(properties)` instead?

Reply:
Good point in principle, I will use `strconv.Quote()`.